CHANGELOG_NEXT.md

```diff
@@ -25,5 +25,6 @@
 
 * chore: remove checks for unsupported node versions [#4341](https://github.com/open-telemetry/opentelemetry-js/pull/4341) @dyladan
 * refactor(sdk-trace-base): remove `BasicTracerProvider._registeredSpanProcessors` private property. [#5134](https://github.com/open-telemetry/opentelemetry-js/pull/5177) @david-luna
+* chore: update typescript to version `^5.6.3` [#5145](https://github.com/open-telemetry/opentelemetry-js/pull/5145) @david-luna
```

Comment:
This almost certainly should move up to the "breaking change" section, no?

Also should that say `5.6.3` rather than `^5.6.3`? The package.json files below (at least some of them) pin the version.

Reply:
At 1st I checked https://github.com/microsoft/TypeScript/wiki/Breaking-Changes and was hesitating since I'm not completely sure how the produced types may break user apps. Also I've tried using `^` in dependencies, its a leftover.

Thanks for spotting it. I'll move it up :)